### ISSUE TITLE
State attributes for predbat.pv_energy exceed maximum size

### ISSUE THREAD

**Issue by MalcolmSpencer** [springfall2008/predbat_addon#6]
I am getting the following warning in the HA log relating to the predbat addon

Core 2024.6.3
Supervisor 2024.06.0
Operating System 12.4
Frontend 20240610.1

Using cached GITHub data for https://api.github.com/repos/springfall2008/batpred/releases age 8.9 minutes
Predbat /config/predbat.py version v8.0.0 currently running, latest version is v8.0.0 latest beta v8.0.0

Logger: homeassistant.components.recorder.db_schema
Source: components/recorder/db_schema.py:598
integration: Recorder ([documentation](https://www.home-assistant.io/integrations/recorder), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+recorder%22))
First occurred: June 18, 2024 at 20:53:18 (1407 occurrences)
Last logged: 16:05:11

    State attributes for predbat.pv_energy exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored
    State attributes for predbat.soc_kw_base10 exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored
    State attributes for predbat.load_power_best exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored
    State attributes for predbat.best_load_energy exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored
    State attributes for predbat.plan_html exceed maximum size of 16384 bytes. This can cause database performance issues; Attributes will not be stored

This is whats in the predbat log file on restart

2024-06-21 06:59:56.884994: Restore saved setting: version = False (was None)
2024-06-21 06:59:56.885047: Restore saved setting: expert_mode = False (was None)
2024-06-21 06:59:56.885080: Restore saved setting: active = False (was None)
2024-06-21 06:59:56.885122: Restore saved setting: pv_metric10_weight = 0.15 (was None)
2024-06-21 06:59:56.885155: Restore saved setting: pv_scaling = 1.0 (was None)
2024-06-21 06:59:56.885186: Restore saved setting: load_scaling = 1.0 (was None)
2024-06-21 06:59:56.885218: Restore saved setting: load_scaling10 = 1.1 (was None)
2024-06-21 06:59:56.885248: Restore saved setting: load_scaling_saving = 1.0 (was None)
2024-06-21 06:59:56.885278: Restore saved setting: battery_rate_max_scaling = 1.0 (was None)
2024-06-21 06:59:56.885308: Restore saved setting: battery_rate_max_scaling_discharge = 1.0 (was None)
2024-06-21 06:59:56.885338: Restore saved setting: battery_loss = 0.03 (was None)
2024-06-21 06:59:56.885370: Restore saved setting: battery_loss_discharge = 0.03 (was None)
2024-06-21 06:59:56.885402: Restore saved setting: inverter_loss = 0.04 (was None)
2024-06-21 06:59:56.885433: Restore saved setting: inverter_hybrid = True (was None)
2024-06-21 06:59:56.885463: Restore saved setting: car_charging_energy_scale = 1.0 (was None)
2024-06-21 06:59:56.885494: Restore saved setting: car_charging_threshold = 6.0 (was None)
2024-06-21 06:59:56.885525: Restore saved setting: car_charging_rate = 7.4 (was None)
2024-06-21 06:59:56.885556: Restore saved setting: car_charging_loss = 0.08 (was None)
2024-06-21 06:59:56.885586: Restore saved setting: best_soc_keep = 0.0 (was None)
2024-06-21 06:59:56.885616: Restore saved setting: set_reserve_min = 4.0 (was None)
2024-06-21 06:59:56.885646: Restore saved setting: car_charging_hold = True (was None)
2024-06-21 06:59:56.885676: Restore saved setting: car_charging_manual_soc = False (was None)
2024-06-21 06:59:56.885705: Restore saved setting: octopus_intelligent_charging = False (was None)
2024-06-21 06:59:56.885734: Restore saved setting: car_charging_plan_smart = False (was None)
2024-06-21 06:59:56.885764: Restore saved setting: car_charging_plan_max_price = 0.0 (was None)
2024-06-21 06:59:56.885792: Restore saved setting: car_charging_from_battery = False (was None)
2024-06-21 06:59:56.885822: Restore saved setting: combine_charge_slots = False (was None)
2024-06-21 06:59:56.885850: Restore saved setting: set_status_notify = True (was None)
2024-06-21 06:59:56.885879: Restore saved setting: set_inverter_notify = False (was None)
2024-06-21 06:59:56.885934: Restore saved setting: set_charge_low_power = False (was None)
2024-06-21 06:59:56.885966: Restore saved setting: set_discharge_during_charge = True (was None)
2024-06-21 06:59:56.885995: Restore saved setting: set_read_only = True (was None)
2024-06-21 06:59:56.886025: Restore saved setting: balance_inverters_enable = False (was None)
2024-06-21 06:59:56.886054: Restore saved setting: debug_enable = False (was None)
2024-06-21 06:59:56.886083: Restore saved setting: car_charging_plan_time = 07:00:00 (was None)
2024-06-21 06:59:56.886111: Restore saved setting: mode = Control charge & discharge (was None)
2024-06-21 06:59:56.886142: Restore saved setting: manual_charge = off (was None)
2024-06-21 06:59:56.886174: Restore saved setting: manual_discharge = off (was None)
2024-06-21 06:59:56.886204: Restore saved setting: manual_idle = off (was None)
2024-06-21 06:59:56.886235: Restore saved setting: manual_freeze_charge = off (was None)
2024-06-21 06:59:56.886264: Restore saved setting: manual_freeze_discharge = off (was None)
2024-06-21 06:59:56.886292: Restore saved setting: auto_update = True (was None)
2024-06-21 06:59:56.886321: Restore saved setting: iboost_enable = True (was None)
2024-06-21 06:59:56.886350: Restore saved setting: carbon_enable = False (was None)
2024-06-21 06:59:56.886379: Restore saved setting: iboost_solar = True (was None)
2024-06-21 06:59:56.886408: Restore saved setting: iboost_gas = False (was None)
2024-06-21 06:59:56.886436: Restore saved setting: iboost_charging = False (was None)
2024-06-21 06:59:56.886465: Restore saved setting: iboost_gas_scale = 1.0 (was None)
2024-06-21 06:59:56.886495: Restore saved setting: iboost_max_energy = 4.0 (was None)
2024-06-21 06:59:56.886525: Restore saved setting: iboost_today = 0.0 (was None)
2024-06-21 06:59:56.886555: Restore saved setting: iboost_max_power = 3000.0 (was None)
2024-06-21 06:59:56.886585: Restore saved setting: iboost_min_power = 500.0 (was None)
2024-06-21 06:59:56.886614: Restore saved setting: iboost_min_soc = 100.0 (was None)
2024-06-21 06:59:56.886646: Restore saved setting: holiday_days_left = 0.0 (was None)

**Comment by github-dpmracing** [comment]
I believe this is detailed in the FAQ and is basically "nothing to see here", you can, if you want to get HA to stop sending the warnings if you wish

[https://springfall2008.github.io/batpred/faq/#predbat-is-causing-warning-messages-about-exceed-maximum-size-in-the-home-assistant-core-log](https://springfall2008.github.io/batpred/faq/#predbat-is-causing-warning-messages-about-exceed-maximum-size-in-the-home-assistant-core-log)
